ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -158,7 +158,7 @@ groupingExpressions
     ;
 
 values
-    : '(' (literal (',' literal)*)? ')'
+    : '(' (expression (',' expression)*)? ')'
```

Comment:
Feels like this would be a better fit:

```suggestion
    : '(' (valueExpression (',' valueExpression)*)? ')'
```

Though I always find the rules for `expression` very confusing...